tests/resources/test_serialized_links.py

```diff
@@ -64,6 +64,7 @@ def test_draft_links(client, draft_json, minimal_record, headers):
         "access_links": f"https://127.0.0.1:5000/api/records/{pid_value}/access/links",  # noqa
         "files": f"https://127.0.0.1:5000/api/records/{pid_value}/draft/files",
         "media_files": f"https://127.0.0.1:5000/api/records/{pid_value}/draft/media-files",
+        "thumbnails": {},
```

Comment:
Question: should the thumbnail be empty or should it not be included in the response if not present?

Reply:
Answer: it shouldn't be included at all if we can't have thumbnails for the record (i.e. no images/PDFs). I guess the `when=...` parameter can handle that.